### ISSUE TITLE
git lfs migrate import error  'encoding/hex: invalid byte: U+0073 's''

### ISSUE THREAD

**Issue by shi-zhenghao** [git-lfs/git-lfs#4709]
**Describe the issue**
When I use git LFS import, the following error is reported in the middle of import.

[root@kube-node1 kmd-generic.git]# git lfs migrate import --everything --above="100MB"
migrate: Sorting commits: ..., done.
encoding/hex: invalid byte: U+0073 's'15/894636)

**System environment**
Centos 8

**Output of `git lfs env`**
git-lfs/2.13.3 (GitHub; linux amd64; go 1.16.2; git a5e65851)
git version 2.32.0

Error reading git config: error running /usr/local/git/libexec/git-core/git 'rev-parse' '--is-bare-repository': 'fatal: not a git repository (or any of the parent directories): .git' 'exit status 128'
LocalWorkingDir=
LocalGitDir=
LocalGitStorageDir=
LocalMediaDir=lfs/objects
LocalReferenceDirs=
TempDir=lfs/tmp
ConcurrentTransfers=8
TusTransfers=false
BasicTransfersOnly=false
SkipDownloadErrors=false
FetchRecentAlways=false
FetchRecentRefsDays=7
FetchRecentCommitsDays=0
FetchRecentRefsIncludeRemotes=true
PruneOffsetDays=3
PruneVerifyRemoteAlways=false
PruneRemoteName=origin
LfsStorageDir=lfs
AccessDownload=none
AccessUpload=none
DownloadTransfers=basic,lfs-standalone-file
UploadTransfers=basic,lfs-standalone-file
GIT_EXEC_PATH=/usr/local/git/libexec/git-core
git config filter.lfs.process = ""
git config filter.lfs.smudge = ""
git config filter.lfs.clean = ""

**Additional context**
I'm using git LFS to migrate repo. I don't know the reason for this problem at all, and I haven't found a similar situation in others. So I can only ask this question for an answer.

**Comment by bk2204** [comment]
Hey,

I think in order to look into this more, we'd need some reproduction steps.  I'm going to mark this accordingly.  If you can provide an example, we'll look into it more.

**Comment by make-all** [comment]
I encountered this trying to analyse a Linux kernel repository. As the size exceeds the 1GB per repository allowed by the git hosting provider, I wanted to see if there was opportunity to move some objects to LFS (which has a 10GB allowance) to get below the 1GB limit for the rest.

Here is the steps to reproduce:

```
git clone https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git
cd linux
git lfs migrate info --include-ref=master
```

**Comment by bk2204** [comment]
Hey,

Thanks for the reproduction case.  I can indeed confirm this on the Linux kernel repository.  I'll look more into why it's failing.

**Comment by squedward** [comment]
I have the same issue on git-lfs/3.2.0. My repository is 57GB, and it is an internal project so I'm not sure how it can be reproduced, but there is no "s" to be found either
```
05:49:39 edward@edbian projects → cat lfs.txt  | grep s
05:49:44 edward@edbian projects → 
```
[lfs.txt](https://github.com/git-lfs/git-lfs/files/9931349/lfs.txt)

**Comment by bk2204** [comment]
It is known and expected that this is broken on Git LFS v3.2.0.  It should be fixed in v3.3.0, which will be released relatively soon (we're waiting on one or two more issues to be closed).  If you build from the main branch using a suitable version of the gitobj library, that will contain the fix.
